### PR TITLE
Unified Cloud Navigation screenshot removal and style edits 

### PR DESCRIPTION
**Pull request by mgarrell777**
### What

This PR is meant to address style and screenshot issues related to the unified cloud navigation effort on certain pages. The pages affected by this PR are:

https://www.terraform.io/cloud-docs/workspaces
https://www.terraform.io/cloud-docs/workspaces/creating
https://www.terraform.io/cloud-docs/workspaces/settings/access
https://www.terraform.io/cloud-docs/workspaces/settings/ssh-keys
https://www.terraform.io/cloud-docs/workspaces/settings/run-triggers
https://www.terraform.io/cloud-docs/workspaces/settings/run-tasks
https://www.terraform.io/cloud-docs/users-teams-organizations/teams
https://www.terraform.io/cloud-docs/users-teams-organizations/permissions
https://www.terraform.io/cloud-docs/users-teams-organizations/2fa
https://www.terraform.io/cloud-docs/users-teams-organizations/single-sign-on
https://www.terraform.io/cloud-docs/users-teams-organizations/single-sign-on/linking-user-account

Additional PRs will be coming to address similar screenshot and other style issue on other Terraform Cloud documentation pages.

### Why

**Language style issues**

Removes references to variations of a "sidebar", a "navigation bar", a "top navigation bar", and other directional language such as "upper right corner". With the UI changes, removing specific references to a navigation bar and specific directional references makes the text more agnostic.

**Removes unnecessary screenshots**

Screenshots are a potential hazard in documentation. Maintaining them is difficult because, as the product changes, screenshots must also change, sometimes drastically or with relative frequency. The maintenance effort places a time and resource burden on both developers and technical writers that takes away from other projects. Screenshots also present potential accessibility challenges, both for disabled users and for potential future translation for non-English speaking users.

Our stance is that screenshots are typically not needed in the documentation because text should clearly, accurately, and succinctly describe processes and results a user may encounter. Our stance is similar to the guidance in the [Google Developers Style Guide](https://developers.google.com/style/images?hl=en), which technical writing uses as a supplement to HashiCorp's own guidance.

If contributors want to include a screenshot, they must provide a rationale explaining why the screenshot provides significant value for users beyond the text. We typically use screenshots when the information in the screenshot is very difficult to explain in words. Screenshots may be helpful when the UI is non-intuitive or especially complex. They may also be useful to show the results of a complex action, like pushing metadata to the UI.

The specific screenshots to be removed have been marked in the text and will be properly removed following review.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages. This requires a separate PR in the [`terraform-website` repository](https://github.com/hashicorp/terraform-website) `redirects.next.js` file.
- [ ] API documentation and the API Changelog have been updated. 
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [ ] I or someone else reviewed the content for technical accuracy.
- [ ] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

File: website/docs/cloud-docs/users-teams-organizations/permissions.mdx [hunk 1]
```diff
@@ -147,7 +147,7 @@ Some permissions - such as the runs permission - are tiered: you can assign one
 
 ## Organization Permissions
 
-Separate from workspace permissions, teams can be granted permissions to manage certain resources or settings across an organization. To set these permissions for a team, click **Settings** in the top navigation bar. Then click **Teams** and select the team name from the list.
+Separate from workspace permissions, teams can be granted permissions to manage certain resources or settings across an organization. To set these permissions for a team, click **Settings**. Then click **Teams** and select the team name from the list.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/users-teams-organizations/permissions.mdx:150]:
```suggestion
Separate from workspace permissions, you can grant teams permissions to manage certain resources or settings across an organization. To set these permissions for a team, go to your organization's **Settings**. Then click **Teams**, and select the team name from the list.
```

File: website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx [hunk 2]
```diff
@@ -46,16 +46,17 @@ If an organization's owners disable SSO (or downgrade the organization's account
...
    ![Screenshot: The Terraform Cloud SSO sign-in page, asking for an organization name.](/img/docs/sign-in.png)
 
 1. If you've signed in to Terraform Cloud with SSO before, proceed to the next step.
 
    If you're signing in for the first time under this account or for the first time accessing this organization, you'll be required to create a new account or link to an existing account. Use the links below the account creation form if you want to link your SSO identity to an existing account, then fill out and submit the relevant form.
-
+**SCREENSHOT FOR DELETION**
    ![Screenshot: Terraform Cloud's create account page.](/img/docs/create-account.png)
-
+**SCREENSHOT FOR DELETION**
```
Comment (laurapacilio) [inline website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx:57]: so something really weird is that I don't see these in the actual docs right now: https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/single-sign-on

hmmm is your branch up-to-date with main?

File: website/docs/cloud-docs/workspaces/index.mdx [hunk 3]
```diff
@@ -48,7 +48,7 @@ Both Terraform Cloud and Terraform CLI have features called workspaces, but they
 
 ## Listing and Filtering Workspaces
 
-Click **Workspaces** in the top navigation bar. Terraform Cloud shows a list of all workspaces in the current organization.
+Click **Workspaces**. Terraform Cloud shows a list of all workspaces in the current organization.
 
 ![Screenshot: the list of workspaces](/img/docs/index-list.png)
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/index.mdx:53]: this is another screenshot that's not in the page that's currently live....
https://developer.hashicorp.com/terraform/cloud-docs/workspaces

File: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx [hunk 4]
```diff
@@ -68,10 +68,12 @@ For example, if a mandatory task fails and an advisory task succeeds, the run fa
 
 The following example shows a run that failed due to a mandatory run task.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/run-tasks.mdx:69]: these I want to discuss. First, if we remove them, then we need to remove the sentences preceding them that are like "the following example shows" 

I think I feel good about taking them out entirely though. I don't know that they're adding much to the user's understanding here.

File: website/docs/cloud-docs/workspaces/settings/run-triggers.mdx [hunk 5]
```diff
@@ -40,10 +43,12 @@ Under the "Source Workspaces" section, select the workspace you would like to co
 
 Runs which are queued in your workspace through a run trigger will include extra information in their run details section. This includes links to the source workspace and the successfully applied run that activated the run trigger.
 
+**SCREENSHOT FOR DELETION**
 ![Screenshot: a run's run details section](/img/docs/run-triggers-run-details.png)
 
 Operators of your source workspaces will be informed of the connection to your workspace during the [plan](/docs/glossary#plan-noun-1-) and [apply](/docs/glossary#apply-noun-) phases of their run.
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/run-triggers.mdx:49]: I was just thinking that maybe if we remove the screenshot we should make the text more specific here. Does this feel like it makes sense to you?

```suggestion
The source workspace includes a message in the [plan](/docs/glossary#plan-noun-1-) and [apply](/docs/glossary#apply-noun-) and apply run details that specifies the workspaces where Terraform Cloud will automatically start runs.

File: website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx [hunk 6]
```diff
@@ -44,10 +45,12 @@ To delete a key, find it in the list of keys and click its "Delete" button. Befo
 
 To assign a key to a workspace, navigate to that workspace's page and choose "SSH Key" from the "Settings" menu.
 
+**SCREENSHOT FOR DELETION**
 ![Terraform Cloud screenshot: the settings menu on a workspace's page](/img/docs/settings-tabs.png)
```
Comment (laurapacilio) [inline website/docs/cloud-docs/workspaces/settings/ssh-keys.mdx:49]: hmmm i don't see this on the actual page that's live right now --> maybe this is me though and I'm missing something!
https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/ssh-keys

**Review by laurapacilio — APPROVED** [review]
I agree with all of the areas where you want to remove screenshots :) nice work!

I had a couple of nitpicks for language (see what you think there) and then in a couple of places....you're wanting to remove screenshots that I don't see reflected in the live docs. So I'm wondering what's going on there. Is it possible your branch isn't up-to-date with `main`?

I see you have some merge conflicts so maybe that's it! lmk if you would like help trouble shooting this. Thank you!

**Comment by mgarrell777** [comment]
Closed because accidentally deleted a necessary file, then re-cloned the repo and deleted the associated local branch. Restarted in [this PR](https://github.com/hashicorp/terraform-docs-common/pull/205)

**Pull request closed without merging**